src/components/SignUp/GeneralInfo.js

```diff
@@ -250,7 +250,7 @@ class GeneralInfo extends Component {
   // Regist general user info.
   regist() {
     const profile = this.state.profile;
-    if (profile.name.replace(/\s/g, '') === '') {
+    if (profile.name === undefined || profile.name.replace(/\s/g, '') === '') {
```

Comment:
Oh! @SarahhhC Thanks!
I'll give you a hint.
You can use `if (!profile.name)` 
It means 'if profile.name is not empty, undefined and null'.
So you can write this code like this.
`if (!profile.name || profile.name.replace(/\s/g, ''))`